# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../openh323/gcc41.patch
# Copyright (C) 2006 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

#These are some includes which are missing from the source tarbal,
#but are present in CVS, at the next upgrade they will be useless

diff -ruN ./src.orig/h261codec.cxx ./src/h261codec.cxx
--- ./src.orig/h261codec.cxx	2006-06-06 11:05:40.000000000 +0300
+++ ./src/h261codec.cxx	2006-09-06 23:25:09.000000000 +0300
@@ -266,6 +266,7 @@
  *
  */
 
+#include <ptlib/video.h>
 #include <ptlib.h>
 
 #ifdef __GNUC__
diff -ruN ./src.orig/h323ep.cxx ./src/h323ep.cxx
--- ./src.orig/h323ep.cxx	2006-06-09 09:30:12.000000000 +0300
+++ ./src/h323ep.cxx	2006-09-06 23:16:21.000000000 +0300
@@ -788,6 +788,7 @@
 #include "../version.h"
 #include "h323pluginmgr.h"
 
+#include <ptlib/sound.h>
 #include <ptclib/random.h>
 #include <ptclib/pstun.h>
 #include <ptclib/url.h>
diff -ruN ./src.orig/h4601.cxx ./src/h4601.cxx
--- ./src.orig/h4601.cxx	2006-06-11 03:17:43.000000000 +0300
+++ ./src/h4601.cxx	2006-09-06 23:22:09.000000000 +0300
@@ -55,6 +55,7 @@
 #pragma implementation "h4601.h"
 #endif
 
+#include <ptlib/pluginmgr.h>
 #include <h460.h>
 #include "h4601.h"
 #include <h323.h>
